io.httpie.HTTPie.yml

```diff
@@ -0,0 +1,75 @@
+app-id: io.httpie.HTTPie
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+separate-locales: false
+tags:
+  - proprietary
+command: httpie
+rename-icon: httpie
+rename-desktop-file: httpie.desktop
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
```

Comment:
missing `--share=ipc` that is necessary for x11.

Reply:
Can you tell the reason why?  I used this: https://github.com/flathub/rest.insomnia.Insomnia/blob/master/rest.insomnia.Insomnia.yml as reference which doesn't use `--share=ipc` . Also, HTTPie is running fine without `--share=ipc`

Reply:
two wrongs don't make a right.

Reply:
from the linter in the build logs:
```
{
    "warnings": [
        "finish-args-contains-both-x11-and-wayland",
        "finish-args-x11-without-ipc"
    ]
}
```

Reply:
Done